documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -27,9 +27,12 @@ The operator generates these event types in a domain namespace, which indicate t
  *  `DomainDeleted`: An existing domain has been deleted.
  *  `DomainProcessingStarting`: The operator has started to process a new domain or to update an existing domain. This event may be a result of a `DomainCreate`, `DomainChanged`, or `DomainDeleted` event, or a result of a retry after a failed attempt.
  *  `DomainProcessingFailed`: The operator has encountered a problem while it was processing the domain resource. The failure either could be a configuration error or a Kubernetes API error.
- *  `DomainProcessingRetrying`: The operator is going to retry the processing of a domain after it encountered an failure.
+ *  `DomainProcessingRetrying`: The operator is going to retry the processing of a domain after it encountered a failure.
  *  `DomainProcessingCompleted`:  The operator successfully completed the processing of a domain resource.
  *  `DomainProcessingAborted`:  The operator stopped processing a domain when the operator encountered a fatal error or a failure that persisted after the specified maximum number of retries.
+ *  `DomainRollStarting`:  The operator has started a rolling restart of a domain after it detects a change or changes in the domain resource or WebLogic domain configuration, which require a restart of all running pods in the domain. The operator makes its best efforts in providing the reason of the rolling restart. If the rolling restart is caused by one or more of the following fields in the domain resource, the event message contains the fields that have changed as well as their old and new values: image, imagePullPolicy, livenessProbe, readinessProbe, domain restartVersion, domainHome, includeServerOutInPodLog, and logHome. When the rolling restart is caused by changes in the other domain resource fields (Please refer to [Fields in domain resource that cause servers to be restarted]({{< relref "/userguide/managing-domains/domain-lifecycle/startup#fields-that-cause-servers-to-be-restarted" >}})), the event message simply indicates that the domain resource has changed. If the cause of the rolling restart is in the WebLogic domain configuration, the event message indicates that as well without the details.
```

Comment:
- MII doc standards require using "then" with every if.
- Hugo allows you to have new lines in text (it strips them out)
- The only WL configuration changes detected are in the Model in Image case
- I think this can be shortened a bit
How about the following?
```
  * The operator has detected domain resource or Model in Image model
    updates that require it to perform a rolling restart.
    If the domain roll is due to a change to domain resource fields
    `image`, `imagePullPolicy`, `livenessProbe`, `readinessProbe`, `restartVersion`,
    `domainHome`, `includeServerOutInPodLog`, or `logHome`, then
    the event message reports the field name plus its old and new values.
    If the domain roll is due to
    [other domain resource changes that cause servers to be restarted]({{< relref "/userguide/managing-domains/domain-lifecycle/startup#fields-that-cause-servers-to-be-restarted" >}})),
    then the event message simply reports that the domain resource has changed.
    If the domain roll is due to a Model in Image model update,
    then the event message reports there has been a change without the details.
```

Reply:
The link contains a full list of the fields that cause a domain to roll. I reworded it a little. Please see if the modified version is fine to you.

Reply:
`The operator has detected domain resource or Model in Image model
    updates that require it to perform a rolling restart of the domain.
    If the domain roll is due to a change to domain resource fields
    `image`, `imagePullPolicy`, `livenessProbe`, `readinessProbe`, `restartVersion`,
    `domainHome`, `includeServerOutInPodLog`, or `logHome`, then
    the event message reports the field name plus its old and new values.
    If the domain roll is due to other domain resource changes that cause servers to be restarted
    (see [full list of fields that cause servers to be restarted]({{< relref "/userguide/managing-domains/domain-lifecycle/startup#fields-that-cause-servers-to-be-restarted" >}})),
    then the event message simply reports that the domain resource has changed.
    If the domain roll is due to a Model in Image model update,
    then the event message reports there has been a change in the WebLogic domain configuration without the details.`

Reply:
Yep, looks very good.

Reply:
LGTM too. Thanks.

Reply:
@tbarnes-us, is this an approval? :)